integreat_cms/cms/views/dashboard/dashboard_view.py

```diff
@@ -42,51 +47,192 @@ def get_context_data(self, **kwargs):
                 ),
             }
         )
-        context.update(self.get_hix_context())
+
+        context.update(self.get_unread_feedback_context())
+        context.update(self.get_unreviewed_pages_context())
+        context.update(self.get_low_hix_value_context())
+        context.update(self.get_outdated_pages_context())
+        context.update(self.get_broken_links_context())
+        context.update(self.get_automatically_saved_pages())
         return context
 
-    def get_hix_context(self):
+    def get_unreviewed_pages_context(self):
         """
-        Extend context by HIX info
+        Extend context by info on unreviewed pages
 
-        :return: The HIX context dictionary
+        :return: Dictionary containing the context for unreviewed pages
         :rtype: dict
         """
-        if not settings.TEXTLAB_API_ENABLED:
-            return {}
 
-        # Get the current region
-        region = self.request.region
-        if not region.hix_enabled:
-            return {}
+        explicitly_archived_ancestors = Page.objects.filter(
+            tree_id=OuterRef("pk"),
+            lft__lt=OuterRef("lft"),
+            rgt__gt=OuterRef("rgt"),
+            explicitly_archived=True,
+        ).values("pk")
+
+        not_implicitly_archived_ids = Page.objects.filter(
+            id=Case(
+                When(
+                    Exists(explicitly_archived_ancestors),
+                    then=None,
+                ),
+                default=F("pk"),
+            )
+        )
```

Comment:
@timobrembeck I played around with this for a bit and the tests don't seem to break overall. Except for the PDF tests because they always make issues, and `test_sitemap` because the query count went down by one:

![image](https://github.com/digitalfabrik/integreat-cms/assets/10530729/fd886255-7393-4af8-90c9-abba9601337a)

Should I add this as a separate PR rather than as part of this one? If so, this one should depend on it and only get merged after it, correct?

Reply:
I committed my suggestion directly to this branch for now.

Reply:
@PeterNerlich wow, that's really cool, awesome improvement! :muscle:
It's fine to include this change in this PR, however could you clean up the git history a bit to have one commit for the dashboard changes and one commit for the refactoring of the model methods?